projects/addon-doc/components/aside/aside.component.html

```diff
@@ -0,0 +1,20 @@
+<ng-content></ng-content>
+<nav
+    tuiTabs
+    vertical="right"
+    [(activeItemIndex)]="activeIndex"
+>
+    <button
+        *ngFor="let item of menuList; let index = index"
+        tuiTab
+    >
+        <a
```

Comment:
You do not need to nest `a` into `button`, `tuiTab` can be applied directly to `a`.

Reply:
but styles doesn't work for a[tuiTab]

<img width="294" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/3147d2b0-d787-4b5f-99d0-0a12612a5085">